ImageLounge/CMakeLists.txt

```diff
@@ -162,8 +162,39 @@ IF(NOT ENABLE_PLUGINS)
 	LIST(REMOVE_ITEM GUI_HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/src/DkPluginManager.h)
 ENDIF(NOT ENABLE_PLUGINS)
 
+# Try to get revision from Git head
+set(NOMACS_REVISION "unknown")
```

Comment:
Instead of "unknown" can we use empty string?

Reply:
It was updated.